spec/factories/ppiu_payment_account.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :ppiu_payment_account, class: 'EVSS::PPIU::PaymentAccount' do
+    account_type 'Checking'
+    financial_institution_name 'Bank of Ad Hoc'
+    account_number '12345678'
```

Comment:
Very minor point, but would there be any value in making the `account_number` a sequence, for variation?  i.e.

```ruby
sequence(:account_number, 10) { |n| "123456#{n}" }
```